From a288bd57433a84b58d87e40b71eb65138313f9aa Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Wed, 19 Nov 2008 13:57:34 -0800
Subject: [PATCH 13/21] Staging: comedi: comedi_pci.h: remove unneeded wrapper

just call pci_disable_device() directly in comedi_pci_enable()
as no wrapper is needed now that we are in the kernel tree.

Cc: David Schleef <ds@schleef.org>
Cc: Frank Mori Hess <fmhess@users.sourceforge.net>
Cc: Ian Abbott <abbotti@mev.co.uk>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/comedi/drivers/comedi_pci.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/comedi/drivers/comedi_pci.h
+++ b/drivers/staging/comedi/drivers/comedi_pci.h
@@ -63,7 +63,7 @@ static inline int comedi_pci_enable(stru
 
 	rc = pci_request_regions(pdev, res_name);
 	if (rc < 0)
-		comedi_pci_disable_no_regions(pdev);
+		pci_disable_device(pdev);
 
 	return rc;
 }
